src/window/vulkan/src/lib.rs

```diff
@@ -291,7 +291,8 @@ impl core::WindowExt<backend::Backend> for Window {
     type Adapter = backend::Adapter;
 
     fn get_surface_and_adapters(&mut self) -> (Surface, Vec<backend::Adapter>) {
-        let instance = Instance::create("gfx-rs demo", 0);
+        let enable_debug = true; //TEMP?
```

Comment:
hmm, maybe enable it for debug builds only? For release it's still possible to activate layers via env variables.